CHANGES.rst

```diff
@@ -1,6 +1,33 @@
 Changes
 =======
 
+Version 0.5.0 (2019-04-19)
+--------------------------
+
+- Improves command line tool documentation by grouping of commands and
```

Comment:
Maybe better: "Improves command line tool help documentation by grouping commands and usage examples." WDYT?